glowfic-dl.py

```diff
@@ -158,13 +253,18 @@ def render_post(post: Tag, image_map: ImageMap) -> RenderedPost:
     post_div = post_html.find("div")
     permalink = post.find("img", title="Permalink", alt="Permalink").parent["href"]
     permalink_fragment = urlparse(permalink).fragment
-    reply_anchor = post_html.new_tag("a", id=permalink_fragment)
+    if permalink_fragment == "":
```

Comment:
I don't think the empty-fragment anchors were actually used anywhere, so if you're going to check for them, is there any reason not to just drop them entirely?

Reply:
None that I can think of! I have now done so.